### DISCUSSION TITLE
Apollo - Accessing context inside new with-apollo example

### DISCUSSION THREAD

**Discussion by hanselabreu** [vercel/next.js#15164] [Help]
Hi all,

Is there any way to access the `context` to get to the `req.headers` inside the new with-apollo example?

The goal would be to pass along those `cookies` within the `ApolloClient` server side to a standalone service.

**Comment by dukesx** [comment]
Yes you can if you use GetServerSideProps and access the context parameter.

Export const getServerSideProps = (context) =>
{
var req = context.req;
}
